curriculum/challenges/english/blocks/zh-a1-learn-certification-introduction/691f071bd6a0ee105c5f571e.md

```diff
@@ -0,0 +1,96 @@
+---
+id: 691f071bd6a0ee105c5f571e
+title: Pinyin-Hanzi Input
+challengeType: 22
+dashedName: pinyin-hanzi-input
+lang: zh-CN
+inputType: pinyin-to-hanzi
+---
+
+<!-- (audio) Wang Hua: 你好 (nǐ hǎo)，世界 (shì jiè)。-->
+
+# --description--
+
+You've already learned how to input correct Pinyin. Now you're ready to input Chinese characters. This is where the Pinyin-Hanzi Input comes in. Try typing `ni3 hao3` and `shi4 jie4` to see how to say "Hello, world" in Chinese!
+
+Again, if you make a mistake, just delete the incorrect part and re-enter it.
+
+# --instructions--
+
+Listen to the audio and complete the sentence below.
+
+# --fillInTheBlank--
+
+## --sentence--
```

Comment:
I'm using a PC. At the moment, I have to copy and paste `nǐ hǎo` and `shì jiè` in the answers for it to work, as typing `ni3 hao3` and `shi4 jie4` did not produce any changes. I'm wondering whether the "How to Set Up the Chinese Keyboard on Your Computer" and "How to Set Up the Chinese Keyboard on Mobile" should come prior to starting filling in the blanks. If they do, they are out of order in the structure file.